test/integration/targets/ansible-playbook-callbacks/all-callbacks.yml

```diff
@@ -96,26 +96,22 @@
       ignore_errors: true
 
     - name: async poll ok
-      command: sleep 2
-      async: 3
-      poll: 1
+      command: sleep 3
+      async: 5
+      poll: 2
 
     - name: async poll failed
-      shell: sleep 2; false
-      async: 3
-      poll: 1
+      shell: sleep 3; false
+      async: 5
+      poll: 2
       ignore_errors: true
 
-    - include_tasks: include_me.yml
```

Comment:
This shouldn't be removed. Add an empty `include_me.yml` file instead.

Reply:
Will do.